code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -447,7 +447,7 @@
 	w_class = 3
 	layer = 4
 
-/obj/item/asteroid/goliath_hide/afterattack(atom/target, mob/user, proximity_flag)
+/obj/item/stack/sheet/animalhide/goliath_hide/afterattack(atom/target, mob/user, proximity_flag)
```

Comment:
It needs some changes here. For now, it doesn't check for amount of hides in the stack, and even if you have 50, they all will be qdelled after the first upgrade.